app/uk/gov/hmrc/cdsreimbursementclaim/services/ccs/CcsSubmissionService.scala

```diff
@@ -77,10 +77,7 @@ class DefaultCcsSubmissionService @Inject() (
       makeBatchFileInterfaceMetaDataPayload(submitClaimRequest, submitClaimResponse)
         .map(data =>
           ccsSubmissionRepo.set(
-            CcsSubmissionRequest(
-              XmlEncoder[Envelope].encode(data),
-              DefaultCcsSubmissionService.getHeaders(hc)
-            )
+            CcsSubmissionRequest(XmlEncoder[Envelope].encode(data), DefaultCcsSubmissionService.getHeaders(hc))
```

Comment:
is this your IDE re-formatting the code no reason at all?

Reply:
Probably

Reply:
it would be good to find the cause as it makes reviewing PRs easier